spec/_layouts/default.html

```diff
@@ -170,11 +170,11 @@ <h1>{{page.title}}</h1>
   			<div class="inner-wrapper">
   				<p>
 FHIR IG © {{site.data.info.copyrightyear}} HL7 &amp; Boston Children's Hospital.
-{{site.data.fhir.packageId}}#{{site.data.fhir.igVer}} based on <a style="color: #81BEF7" href="{{site.data.fhir.path}}">FHIR {{site.data.fhir.version}}</a>.
+{{site.data.fhir.packageId}}#{{site.data.fhir.igVer}} based on <a style="color: #81BEF7" href="http://www.hl7.org/fhir/R4/">FHIR {{site.data.fhir.version}}</a>.
 Generated {{site.data.fhir.genDay}}
           <span style="color: #FFFF77"> |
             <a style="color: #81BEF7" target="_blank" href="http://hl7.org/fhir/bulkdata/history.cfml">History</a>
-            | <a style="color: #81BEF7" target="_blank" href="http://hl7.org/fhir-issues">Propose a change</a>
+            | <a style="color: #81BEF7" target="_blank" href="https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemBrowse&amp;tracker_id=677">Propose a change</a>
```

Comment:
This needs to be http://hl7.org/fhir-issues not GForge, because we'll be transitioning from GForge to Jira soon.